redisinsight/ui/src/packages/redisearch/src/components/TableResult/TableResult.tsx

```diff
@@ -77,6 +77,7 @@ const TableResult = React.memo((props: Props) => {
             <EuiToolTip
               position="bottom"
               title={title}
+              className="text-multiline-ellipsis"
               anchorClassName={cx('tooltip')}
               content={formatLongName(value.toString())}
```

Comment:
Alternative approach is changing the arguments of `formatLongName` here. However, I like the css implementation more.